{"sections":[],"variants":[{"paths":["\/documentation\/discord\/discordgatewayclosereason\/invalidsequence"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/Discord\/documentation\/Discord\/DiscordGatewayCloseReason\/invalidSequence","interfaceLanguage":"swift"},"kind":"symbol","hierarchy":{"paths":[["doc:\/\/Discord\/documentation\/Discord","doc:\/\/Discord\/documentation\/Discord\/DiscordGatewayCloseReason"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["Linux"],"tokens":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"invalidSequence","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:7Discord0A18GatewayCloseReasonV","text":"DiscordGatewayCloseReason","kind":"typeIdentifier","identifier":"doc:\/\/Discord\/documentation\/Discord\/DiscordGatewayCloseReason"}]}]}],"metadata":{"symbolKind":"property","roleHeading":"Type Property","externalID":"s:7Discord0A18GatewayCloseReasonV15invalidSequenceACvpZ","fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"invalidSequence"},{"kind":"text","text":": "},{"preciseIdentifier":"s:7Discord0A18GatewayCloseReasonV","kind":"typeIdentifier","text":"DiscordGatewayCloseReason"}],"modules":[{"name":"Discord"}],"role":"symbol","title":"invalidSequence"},"schemaVersion":{"major":0,"minor":2,"patch":0},"abstract":[{"type":"text","text":"We sent a bad sequence number when trying to resume."}],"references":{"doc://Discord/documentation/Discord/DiscordGatewayCloseReason/invalidSequence":{"role":"symbol","type":"topic","identifier":"doc:\/\/Discord\/documentation\/Discord\/DiscordGatewayCloseReason\/invalidSequence","title":"invalidSequence","kind":"symbol","url":"\/documentation\/discord\/discordgatewayclosereason\/invalidsequence","fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"invalidSequence"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"DiscordGatewayCloseReason","preciseIdentifier":"s:7Discord0A18GatewayCloseReasonV"}],"abstract":[{"type":"text","text":"We sent a bad sequence number when trying to resume."}]},"doc://Discord/documentation/Discord/DiscordGatewayCloseReason":{"role":"symbol","type":"topic","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"DiscordGatewayCloseReason"}],"identifier":"doc:\/\/Discord\/documentation\/Discord\/DiscordGatewayCloseReason","url":"\/documentation\/discord\/discordgatewayclosereason","navigatorTitle":[{"kind":"identifier","text":"DiscordGatewayCloseReason"}],"kind":"symbol","title":"DiscordGatewayCloseReason","abstract":[{"text":"Represents the reason a gateway was closed.","type":"text"}]},"doc://Discord/documentation/Discord":{"role":"collection","kind":"symbol","type":"topic","identifier":"doc:\/\/Discord\/documentation\/Discord","abstract":[],"title":"Discord","url":"\/documentation\/discord"}}}